contracts/governance/Staking/WeightedStaking.sol

```diff
@@ -382,17 +382,19 @@ contract WeightedStaking is Checkpoints {
      * @param account The address of the account to check.
      * @param date The lock date.
      * @param blockNumber The block number to get the vote balance at.
+     * @param isDirectGovernanceWithdraw Flag for direct governance withdrawal.
      * @return The number of votes the account had as of the given block.
      * */
     function getPriorUserStakeByDate(
         address account,
         uint256 date,
-        uint256 blockNumber
-    ) external view returns (uint96) {
+        uint256 blockNumber,
+        bool isDirectGovernanceWithdraw
```

Comment:
to make the name more universal else tightly bound to only one operation
```suggestion
    bool calcForVestingContract
```

Reply:
This param has been removed to support backward compatibility, [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/892d1be815121b95033677706a1c9cb5677c31e7)